index.html

```diff
@@ -172,8 +172,7 @@ <h1 class="project-name">neveragain.tech</h1>
 <li><a href="https://diracdeltas.github.io">Yan Zhu, Security Engineer</a>
 
 <!-- Append an <li> and your own name and position/affiliation below. -->
-
-  
+<li>Jen-Mei Wu, <a href="http://oaklandmakerspace.com/">Liberating Ourselves Locally</a>
```

Comment:
Hey there, we realized we need to do these in alphabetical order to minimize conflicts. Can you update this? Let me know if you need any help figuring out how to rebase / amend.

See https://github.com/neveragaindottech/neveragaindottech.github.io/pull/23 for updated README instructions.